js/README.md

```diff
@@ -11,7 +11,7 @@ Install it with: `npm install twitter-text`
 The `twttr.txt` namespace is exported, making it available as such:
 
 ``` js
-var twitter = require('twitter-text')
+var twitter = require('twitter-txt')
```

Comment:
Hi @montogeek. I wasn't able reproduce this problem. 

'twitter-text' seems to be the correct npm module name.